weather_server/show_weather/views.py

```diff
@@ -75,4 +76,22 @@ def csv_observation_request(request, station, start, end):
         )
     )
     response.seek(0)
-    return HttpResponse(response)
+    return HttpResponse(response, content_type="text/html")
```

Comment:
We don't actually have to specify the ``content_type`` here. From [the Django docs](https://docs.djangoproject.com/en/1.8/ref/request-response/#id3):

> content_type is the MIME type optionally completed by a character set encoding and is used to fill the HTTP Content-Type header. If not specified, it is formed by the DEFAULT_CONTENT_TYPE and DEFAULT_CHARSET settings, by default: “text/html; charset=utf-8”.

Also, since its type is not ``text/html`` but in reality ``text/plain``, IMHO it's better to leave it blank. Downloading works just fine and specifying it as plain isn't a big win, it just makes the code a bit more verbose.